kinappserver/models/__init__.py

```diff
@@ -16,3 +16,4 @@
 from .blacklisted_phone_numbers import *
 from .system_config import *
 from .category import *
+from .app_descovery import *
```

Comment:
@katzio the file name is still app_descovery .... pls fix

Reply:
wut, It's not